eva/models/storage/batch.py

```diff
@@ -249,18 +318,90 @@ def concat(cls, batch_list: Iterable[Batch], copy=True) -> Batch:
 
         return Batch(frame)
 
+    @classmethod
+    def inner_join(cls, first: Batch, second: Batch) -> Batch:
```

Comment:
Can we make it join and pass `inner` as an argument? We will need `left` join in an upcoming feature.